migrations/20231102101356_user.py

```diff
@@ -4,17 +4,51 @@
 # Author: Jeny Sadadia <jeny.sadadia@collabora.com>
 
 """
-Migration for User schema
+Migration for User schema after the adoption of fastapi-users for
+user management in commit:
+
+    api.main: update/add user management routes
+
 """
 
 name = '20231102101356_user'
-dependencies = ['20221014061654_set_timeout']
+dependencies = []
+
+def user_upgrade_needed(user):
```

Comment:
Even though there is no restriction on function names returning boolean values, following the `is_*` pattern would help developers easily predict its return value.
```suggestion
def is_upgrade_needed(user):
```
I removed `user` from the name as this migration file is specific to user migrations.

Reply:
Okay. That was just a suggestion :+1: